src/game/game.cpp

```diff
@@ -1167,7 +1167,6 @@ bool Game::removeCreature(std::shared_ptr<Creature> creature, bool isLogout /* =
 		size_t i = 0;
 		for (const auto &spectator : playersSpectators) {
 			if (const auto &player = spectator->getPlayer()) {
-				player->sendMagicEffect(tilePosition, CONST_ME_POFF);
```

Comment:
This not affect on the logout? could you validate something like `creature->health <= 0`?

Reply:
You're right.

Reply:
I removed the line and the POFF effect continues to be displayed at logout, I believe it is in another location. Just in case, the function has an isLogout parameter, I will include a check for this variable

Reply:
You can keep it as before, canary handles the logout here 👀 
https://github.com/opentibiabr/canary/blob/3cd4031bd746e40dd88967b6726c6392ddf00955/src/server/network/protocol/protocolgame.cpp#L693-L695

Reply:
![cat](https://github.com/user-attachments/assets/5be8d460-e852-4110-a673-683335e424ba)
Okay, I'll update here!